app/src/main/java/at/irfc/app/ui/core/ZoomableImage.kt

```diff
@@ -0,0 +1,60 @@
+package at.irfc.app.ui.core
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.gestures.detectTransformGestures
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.layout.onSizeChanged
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.IntSize
+import at.irfc.app.R
+
+@Composable
+fun zoomableImage(minScale: Float, maxScale: Float, imageId: Int) {
```

Comment:
```suggestion
fun ZoomableImage(minScale: Float, maxScale: Float, painter: Painter, contentDescription: String?) {
```

Naming convention: Composables start with an uppercase letter.

parameters for painter and contentDescription are needed to keep it "generic"